### ISSUE TITLE
Make trade box indicate the waiting better when you accept someone's trade offer

### ISSUE THREAD

**Issue by elancast** [ctriolo/settle#78]
Sorry for wordiness of title...

So I'm referring to the situation when:
1. it's person B's turn
2. B offers person A a trade
3. A gets a dialogue to accept
4. A accepts
5. A's dialogue turns green, B gets a dialogue to accept
6. B accepts
7. Trade over; A and B go back to normal

My problem is with step 5 for person A. Their dialogue box turns green, but the buttons (accept, reject) are still enabled. For me as the unwary user A, I thought the trade was done (so step 6 didn't have to happen) and was wondering why there was a delay with the green dialogue box.

Suggested solutions:
-make it 2way handshake rather than 3 (so cut out step 6)
-remove accept/reject buttons in 5 for A and instead have a disabled "waiting for <person> to accept" button

**Comment by dcapra12** [comment]
makes sense to me. Can't be a 2way handshake, cause the original offerer needs to be able to choose which accept to accept (if that makes sense). Waiting for <name> to accept sounds like a plan.
